includes/admin/class-wc-admin-menus.php

```diff
@@ -116,6 +106,34 @@ public function settings_page_init() {
 		do_action( 'woocommerce_settings_page_init' );
 	}
 
+	/**
+	 * Handle saving of settings.
+	 *
+	 * @return void
+	 */
+	public function save_settings() {
```

Comment:
Great that the method for saving the settings is now extracted. Much better than hidden inside an unrelated method.